kroxylicious-filters/kroxylicious-record-validation/src/main/java/io/kroxylicious/proxy/filter/validation/validators/bytebuf/JsonSchemaBytebufValidator.java

```diff
@@ -59,7 +62,12 @@ public CompletionStage<Result> validate(ByteBuffer buffer, Record record, boolea
                     : CompletableFuture.completedFuture(new Result(false, jsonValidationResult.toString()));
         }
         catch (RuntimeException ex) {
-            return CompletableFuture.completedStage(new Result(false, ex.getMessage()));
+            if (ex.getCause() instanceof Error error) {
```

Comment:
`Error` made me do a double take! I thought it was something to do with java.lang.Error at first.  Phew!

Reply:
No, we're leaking Apicurio implementation details here into Kroxy, I'll take that back.